plugins/woocommerce/package.json

```diff
@@ -128,14 +128,14 @@
 					"testEnv": {
 						"start": "env:test",
 						"config": {
-							"phpVersion": "8.1",
+							"phpVersion": "8.2",
 							"wpVersion": "latest"
 						}
 					},
 					"events": [
 						"pull_request",
 						"push",
-						"php-unit-php-8.8-wp-latest"
+						"php-unit-php-8.2-wp-latest"
```

Comment:
@adimoldovan: not sure about this event, should I update it somewhere else (search point to no original value)?

Reply:
Nope, it's not referenced anywhere. It only exists for convenience, in case one wants to run only this job on demand.

Reply:
Ok, just updating it then.